utils/str.c

```diff
@@ -87,3 +87,28 @@ char *pv_str_skip_prefix(char *str, const char *key)
 	}
 	return str;
 }
+
+size_t epochsecstring(char *buf, size_t len, time_t t)
+{
+	struct tm *tmp;
+	size_t ret;
+	if (!buf)
+		return ~0;
```

Comment:
just a question, what is the ~ doing in the returns?

Reply:
I wanted ot return bitwise not 1 as a special value for errors. I guess I did that because I don't like the ambiguity of 0 being an error code and a return code on success. I am changing it again to 0, let's keep it in line with standard library.

Reply:
ok thanks! rest of the change looks good to me